specification/gedcom-02-datatypes.md

```diff
@@ -88,7 +88,9 @@ epoch   = %s"BCE" / extTag ; constrained by calendar
 
 In addition to the constraints above:
 
-- The allowable `month`s and `epoch`s are determined by the `calendar`.
+- The allowable `day`s, `month`s, `year`s, and `epoch`s are determined by the `calendar`.
+    All known calendars restrict `day` to be between 1 and a month-specific maximum.
+    The largest known maximum is 36, and most months in most calendars have a lower maximum.
```

Comment:
This text allows for a calendar specific rule, although it would be nice to be more explicit about how to handle an unrecognized calendar.  E.g., "If the calendar is unrecognized, implementations can assume that the day is restricted to a value between 1 and 36."